cloudinit/config/schema.py

```diff
@@ -346,8 +397,10 @@ def get_parser(parser=None):
             description='Validate cloud-config files or document schema')
     parser.add_argument('-c', '--config-file',
                         help='Path of the cloud-config yaml file to validate')
-    parser.add_argument('-d', '--doc', action="store_true", default=False,
-                        help='Print schema documentation')
+    parser.add_argument('-d', '--doc', action='store_true',
+                        help='Print specified schema module docs.')
```

Comment:
I'm confused,  what's the difference between --doc and --module ?

Reply:
I see now this doesn't make sense. I had wanted to allow --doc users to also specify --module to limit the docs that were renderered on the CLI. Instead I have consolidated --doc and --module params to to just `--docs`, which can be either 'all' or some '<cc_name>' if we want to limit docs to a single config module's documentation.

Reply:
+1 sounds good.